tests/ingest_runs/test_genomic_data_loader.py

```diff
@@ -0,0 +1,396 @@
+    expected_columns = {
+        CONCEPT.SEQUENCING.TARGET_SERVICE_ID,
+        CONCEPT.SEQUENCING.CENTER.TARGET_SERVICE_ID,
+        CONCEPT.SEQUENCING.ID,
+        CONCEPT.GENOMIC_FILE.SOURCE_FILE,
+    }
+    for col in expected_columns:
+        assert col in output
+    assert output.shape[1] == len(expected_columns)
+    gfs = study_generator.dataservice_payloads["genomic_file"]
+    harm_gf = [
+        key for key, value in gfs.items() if value["is_harmonized"] == "False"
+    ]
+    assert output.shape[0] == len(harm_gf)
+
+
+def test_load_seq_exp_harmonized_genomic_files(
+    study_generator, mock_s3_scrape, mock_load_entities, mock_get_entities,
+    mock_validate
+):
+    """
+    Test the _load_seq_exp_harmonized_genomic_files_ function. Here no steps of
+    the ingest process are mocked, so this is really testing the entire
+    process.
+    """
+    loader = GenomicDataLoader(
+        study_id=study_generator.study_id, init_logger=True
+    )
+    manifest_df = study_generator.dataframes['gwo_manifest.tsv']
+    output = loader.ingest_gwo(manifest_df)
+    mock_s3_scrape.assert_called_once()
+
+    # Check that _load_entities_ was called the right number of times with the
+    # correct arguments.
+    assert mock_load_entities.call_count == 3
+    # First genomic-files are loaded correctly
+    entity_type1, df1 = mock_load_entities.call_args_list[0][0]
+    assert type(df1) == pd.DataFrame
+    assert entity_type1 == "genomic_file"
+    assert not df1.empty
+    manifest_df = manifest_df.drop_duplicates(["KF Participant ID"])
+    assert manifest_df.shape[0] == df1.shape[0]
+    # Check that the columns are as expected
+    expected_columns1 = {
+        CONCEPT.BIOSPECIMEN.TARGET_SERVICE_ID,
+        CONCEPT.GENOMIC_FILE.DATA_TYPE,
+        CONCEPT.GENOMIC_FILE.FILE_NAME,
+        CONCEPT.GENOMIC_FILE.HASH_DICT,
+        CONCEPT.GENOMIC_FILE.SIZE,
+        CONCEPT.GENOMIC_FILE.URL_LIST,
+        CONCEPT.GENOMIC_FILE.FILE_FORMAT,
+        CONCEPT.GENOMIC_FILE.ID,
+        CONCEPT.GENOMIC_FILE.SOURCE_FILE,
+        CONCEPT.GENOMIC_FILE.HARMONIZED,
+        CONCEPT.GENOMIC_FILE.VISIBLE,
+    }
+    for column in expected_columns1:
+        assert column in df1
+    assert df1.shape[1] == len(expected_columns1)
+    # Check biospecimen-genomic-files are loaded correctly
+    entity_type2, df2 = mock_load_entities.call_args_list[1][0]
+    assert entity_type2 == "biospecimen_genomic_file"
+    assert type(df2) == pd.DataFrame
+    assert not df2.empty
+    assert manifest_df.shape[0] == df2.shape[0]
+    expected_columns2 = {
+        CONCEPT.GENOMIC_FILE.ID,
+        CONCEPT.BIOSPECIMEN.TARGET_SERVICE_ID,
+        CONCEPT.GENOMIC_FILE.TARGET_SERVICE_ID,
+        CONCEPT.BIOSPECIMEN.ID,
+        CONCEPT.BIOSPECIMEN_GENOMIC_FILE.VISIBLE,
+    }
+    for col in expected_columns2:
+        assert col in df2
+    assert df2.shape[1] == len(expected_columns2)
+    # Check sequencing-experiment-genomic-files for the harmonized
```

Comment:
Similar to a previous comment. This stuff should probably in its own test for the `load_seq_exp_harmonized_genomic_files` method where the test setup would simply be:

```
def test_load_seq_exp_harmonized_genomic_files(
    study_generator, mock_s3_scrape, mock_load_entities, 
):
    loader = GenomicDataLoader(
        study_id=study_generator.study_id, init_logger=True
    )
    manifest_df = study_generator.dataframes['gwo_manifest.tsv']
    df = loader.ingest_gwo(manifest_df)
    # All the checks you have below this line would be heree
```